cms/djangoapps/contentstore/views/tests/test_course_index.py

```diff
@@ -479,28 +479,26 @@ def test_verify_deprecated_warning_message_with_single_feature(self, publish):
         Verify deprecated warning info for single deprecated feature.
         """
         block_types = ['notes']
-        with override_settings(DEPRECATED_BLOCK_TYPES=block_types):
-            course_module = modulestore().get_item(self.course.location)
-            self._create_test_data(course_module, create_blocks=True, block_types=block_types, publish=publish)
-            info = _deprecated_blocks_info(course_module, block_types)
-            self._verify_deprecated_info(
-                course_module.id,
-                course_module.advanced_modules,
-                info,
-                block_types
-            )
+        course_module = modulestore().get_item(self.course.location)
```

Comment:
Add note about why settings are unnecessary, change to ddt to combine with next test case.